setup.py

```diff
@@ -18,7 +18,7 @@
 requirements = [
         "bitarray-hardbyte>=0.8.0",
         "cffi>=1.7",
-        "clkhash~=0.14",
+        "clkhash~=0.15",
```

Comment:
what does ~= do? This is not described in setuptools https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies

Reply:
That's a good question. I have not even thought about it and just updated the value... From some documentation, it is equivalent to `>=0.15 and ==0.*`, which I should update to `~=0.15.0` instead.
https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
ah, ok. It just seems somewhat inconsistent that we are fine with '>=' for all other dependencies, but not clkhash.
More importantly though, a breaking change in clkhash will cause the anonlink tests to fail. You go from a perfectly fine state of master to broken without changing any code. But actually, anonlink is still fine as ever and good to use.
Thus, I would hard pin the clkhash version to something that works with the tests.

Reply:
With the version set as ~=0.15.0, this means it will accept all versions greater or equal than 0.15.0 but only in the 0.15.* range which I would assume does not have breaking changes, otherwise we should go to 0.16.